src/app/mypage/components/profile/index.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { useState, ChangeEvent } from 'react';
+import { Input } from '@/components/Common';
+import { getProfileImageById } from '@/app/lib/dummyData';
+import { FileEdit, FileThumbnails, useReadFile } from '../Common';
+
+export default function Profile() {
+  const [profile, setProfile] = useReadFile(
+    getProfileImageById(1),
+    'profile.png',
+    'image/png',
+  );
+  const [name, setName] = useState('');
+
+  const handleNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setName(e.target.value);
+  };
+
+  return (
+    <section className="flex flex-col items-center justify-center pb-250 gap-100">
+      <p className="text-30 font-semibold py-20">Profile</p>
+      <article className="flex flex-row w-full px-100 items-center gap-80">
+        <div className="flex flex-col items-center gap-10">
+          <FileThumbnails
+            file={profile}
+            deleteFileHandler={() => setProfile(null)}
+          />
+          <FileEdit setFile={setProfile} />
+        </div>
+        <div className="flex flex-col">
+          <p className="text-[#747373]">Name</p>
+          <Input
+            id="name"
+            type="text"
+            value={name}
+            onChange={handleNameChange}
```

Comment:
공통 컴포넌트인 Input 컴포넌트 내부에서 e.target.value를 할당해서 사용하고 있습니다.
`handleNameChange`를 사용하기 보다 `onValueChange`에 `setName`를 직접 전달해도 돼요.

```suggestion
            onValueChagne={setName}
```